app/shared/video-player/src/commonMain/kotlin/ui/state/PlayerState.kt

```diff
@@ -155,8 +156,17 @@ interface PlayerState {
     val audioTracks: TrackGroup<AudioTrack>
 
     fun saveScreenshotFile(filename: String)
+
+    val chapters: StateFlow<MutableList<Chapter>>
 }
 
+@Immutable
+data class Chapter(
+    val name: String,
+    val duration: Long,
+    val offset: Long
```

Comment:
备注一下单位, 最好是直接把单位带名字里:
```suggestion
    val durationMillis: Long,
    val offsetMillis: Long
```